src/rrdp.rs

```diff
@@ -172,8 +242,270 @@ impl NotificationFile {
             _ => Err(XmlError::Malformed)
         }
     }
+
+    /// Turn into RFC 8182 XML
+    pub fn to_xml<W: io::Write>(&self, writer: &mut Writer<W>) -> Result<(), ProcessError> {
```

Comment:
I think `write_xml` might be a better name to point out that you are not creating some sort of DOM object in memory.

Reply:
not applicable anymore